cyclonedx-lib/build.xml

```diff
@@ -21,7 +21,7 @@
 -->
 
         <!-- Dependency versions -->
-        <property name="cyclonedx-core-java-version" value="7.1.4"/>
+        <property name="cyclonedx-core-java-version" value="7.3.2"/>
         <property name="jackson-core-version" value="2.13.3"/>
```

Comment:
should be at 2.14.2 - maybe check all deps here

Reply:
@julian55455 See: https://github.com/CycloneDX/cyclonedx-core-java/blob/6ad9664733c418eea9729490bab5ea74f0c1aa03/pom.xml#L147